js/gratipay/notification.js

```diff
@@ -5,11 +5,13 @@
 Gratipay.notification = function(text, type, timeout, closeCallback) {
     var type = type || 'notice';
     var timeout = timeout || (type == 'error' ? 10000 : 5000);
-
-    var dialog = ['div', { 'class': 'notification notification-' + type }, [ 'div', text ]];
+    var id = Math.random().toString(36).substring(2, 100);
+    var placeholder = ['div', {'class': 'notification notification-' + type}, ['div', text]];
```

Comment:
Yeah, I had to unravel this while working here. Better with a9caa084917353b000ce35d42c91588d5c8a090b?